mkt/ratings/views.py

```diff
@@ -27,25 +26,16 @@
 log = commonware.log.getLogger('z.api')
 
 
-class RatingPaginator(Paginator):
-    # FIXME: This is only right when ?app= filtering is applied, if no
-    # filtering or if ?user= filtering is done, it's completely wrong.
-    @property
-    def count(self):
```

Comment:
`count` is used 4 times on the serializer. (The default paginator uses `count` to calculate the results once). This means that 4 times, we loaded the whole object list, and 4 times, we loaded the related `addon` instance. Because the our models have transformers loading extra stuff, that's roughly 25 extra queries *per review* in the list.